src/DataRootTuple.sol

```diff
@@ -11,4 +11,6 @@ struct DataRootTuple {
     uint256 height;
     // Data root.
     bytes32 dataRoot;
+    // Celestia original block size.
+    uint256 blockSize;
```

Comment:
square size or block size? can we get more docs on the units or what this is?

Reply:
good catch, will update to square size

Reply:
[07f464e](https://github.com/celestiaorg/quantum-gravity-bridge/pull/145/commits/07f464e7196198999999c2e12ff78247a78ba36b)